lms/djangoapps/program_enrollments/api/linking.py

```diff
@@ -177,35 +145,37 @@ def _get_lms_users(lms_usernames):
     }
 
 
-def _link_program_enrollment(program_enrollment, user):
+def link_program_enrollment_to_lms_user(program_enrollment, user):
```

Comment:
This function name is way too similar to `link_program_enrollments_to_lms_user`. Maybe update the internal single item function to `link_single_program_enrollment_to_lms_user` or something less verbose but distinguishable?

Reply:
Good point. I now have:
* `link_program_enrollments` for multiple
* `link_program_enrollment_to_lms_user` for single.

Let me know if that sounds good.